src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -64,7 +64,7 @@
       <remarks>
         <para>
           This transform uses a set of aggregators to count the number of non-default values for each slot and
-          instantiates a <see cref="SlotsDroppingTransformer"/> to actually drop the slots.
+          instantiates a <see cref="T:Microsoft.ML.Transforms.SlotsDroppingTransformer"/> to actually drop the slots.
```

Comment:
>Microsoft.ML.Transforms [](start = 38, length = 23)

Similar question, not sure why this suddenly became necessary when previously it was not. Or did we only just realize it was necessary? Totally fine, consistent with everythign else I see here, just a little odd.

Reply:
this was not necessary; spotted the difference when changed the namespaces below, and i guess my OCD took over :)

---
In reply to: [260843071](https://github.com/dotnet/machinelearning/pull/2755#discussion_r260843071) [](ancestors = 260843071)